dolphin/drivers/dellemc_storage/client.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 The SODA Authors.
```

Comment:
suggesting folder name change , can we keep this in /drivers/dell_emc/vmax ? because each vendor will have different models of storages

Reply:
Done